src/api/requestBuilder.ts

```diff
@@ -20,18 +20,17 @@ export interface Request {
     body?: JsonBody;
 }
 
-export interface ApiVariety {
-    barong: string;
-    applogic: string;
-    peatio: string;
-}
+const getAPI = () => {
+    const hostUrl = window.location.hostname === 'localhost' ? 'http://localhost:9002' : '';
 
-const getAPI = () => ({
-    barong: authUrl(),
-    applogic: applogicUrl(),
-    peatio: tradeUrl(),
-    finex: finexUrl(),
-});
+    return {
+        barong: authUrl(),
+        applogic: applogicUrl(),
+        peatio: tradeUrl(),
+        finex: finexUrl(),
+        sonic: `${hostUrl}/api/v2`,
```

Comment:
you can use hostUrl for anothe APIs

Reply:
for now if you need this you can use `apiVersion: 'sonic'`